jumpcloud-ADMU/Powershell/ProgressForm.ps1

```diff
@@ -0,0 +1,282 @@
+function DecodeBase64Image {
```

Comment:
When we build the EXE, technically this function is duplicated in the ADMU.ps1 file.

I'm going to write a card to address this before we release 2.7.0.

Reply:
Agree, was thinking of commenting it out for local testing purposes in the future.